src/main/scala/uk/gov/hmrc/wco/dec/Declaration.scala

```diff
@@ -66,25 +68,21 @@ trait JacksonMapper {
 @JsonIgnoreProperties(Array("_xml", "_schema", "_props"))
 @JacksonXmlRootElement(namespace = NS.dms, localName = "MetaData")
 case class MetaData(@JacksonXmlProperty(localName = "WCODataModelVersionCode", namespace = NS.dms)
-                    wcoDataModelVersionCode: Option[String] = None, // max 6 chars
-
+                    wcoDataModelVersionCode: Option[String] = None,
```

Comment:
Is there any reason why the comments/notes around field length constraints have been removed? I was kinda keeping these as a handy reference for when we implement validators. If the response schema has added to the info needed here, then please add it.